.gitignore

```diff
@@ -86,3 +86,6 @@ TAGS
 
 # pyenv
 .python-version
+
+# .pkl files in tests folder
+ tests/*.pickle.*
```

Comment:
Could you move this into `tests/.gitignore`?

Also, having text files without a trailing LF is a bad tone.